content/attacks/posts/2023-11-23-HecoBridge.md

```diff
@@ -20,9 +20,9 @@ On November 22, 2023, Heco Bridge and HTX Exchange were victims of cyberattacks,
 
 ## Attackers
 
-The attacker has yet to be identified but used the following wallets to transfer some of the high-jacked funds:
+The attacker has yet to be identified but used the following wallets to transfer some of the hijacked funds:
```

Comment:
Please, rephase the sentence. I would change 'hijacked' term to 'stolen' or 'drained'.